### PR TITLE
R_rainclouds.R

### PR DESCRIPTION
**Pull request by orchid00**
style R_rainclouds.R

### CODE REVIEW

**Comment by KirstieJane** [comment]
Hi @orchid00!! I am SO sorry for leaving this abandoned for so long!

I'm not an `R` person, and our `R` people aren't used to working on github so we've ended up not making time to review your updates. I'm really sorry about that.

I've downloaded your version of the `.R` files and sent them to Rogier to review.

In terms of the merge conflicts, my plan is to just accept all your changes, but I'll give you a couple of days to have a look just in case there have been updates to master that you'd want to do differently!

Thank you again SO SO much, and, again, I'm really sorry for the delay in getting back to you.

**Comment by orchid00** [comment]
hi @KirstieJane 

I've fixed the conflicts, 
no need to say sorry, I understand everyone is busy :)

File: tutorial_R/R_rainclouds.R [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+### and the package ggplot2 by Hadley Wickham
 
-#Load packages
+# Check if required packages are installed ----
+packages <- c("cowplot", "readr", "ggplot2", "dplyr", "lavaan", "smooth", "Hmisc")
+if (length(setdiff(packages, rownames(installed.packages()))) > 0) {
+  install.packages(setdiff(packages, rownames(installed.packages())))
+}
+
+# Load packages ----
```
Comment (KirstieJane) [inline tutorial_R/R_rainclouds.R]: This is so fun that you've helped us out with this! Thank you!

File: tutorial_R/simulateData.R [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
-simdat<-c(simdat_group1,simdat_group2)
-simdat<-data.frame(c(rep('Group1',times=n),rep('Group2',times=n)),simdat)
-colnames(simdat)<-c('group', 'score')
-
-# calculate summary stats
-
-summary_simdat <- summarySE(simdat, measurevar="score", groupvars=c("group"))
+# Calculate summary stats ----
+summary_simdat <- summarySE(simdat, measurevar = "score", 
+                                     groupvars = c("group"))
```
Comment (KirstieJane) [inline tutorial_R/simulateData.R]: This file looks great to me ✨

File: tutorial_R/summarySE.R [hunk 3]
```diff
@@ -1,48 +1,57 @@
...
-  # Calculate t-statistic for confidence interval: 
+  # Calculate t-statistic for confidence interval:
   # e.g., if conf.interval is .95, use .975 (above/below), and use df=N-1
-  ciMult <- qt(conf.interval/2 + .5, datac$N-1)
+  ciMult <- qt(conf.interval / 2 + .5, datac$N - 1)
   datac$ci <- datac$se * ciMult
-  
+
   return(datac)
 }
```
Comment (KirstieJane) [inline tutorial_R/summarySE.R]: All fab! Thank you! 🚀

File: tutorial_R/R_rainclouds.R (hunk 1 above)
Comment (orchid00) [reply tutorial_R/R_rainclouds.R]: :)

File: tutorial_R/summarySE.R (hunk 3 above)
Comment (orchid00) [reply tutorial_R/summarySE.R]: yay!

File: tutorial_R/simulateData.R (hunk 2 above)
Comment (orchid00) [reply tutorial_R/simulateData.R]: :star2:

**Comment by orchid00** [comment]
great! yay! happy I've helped a bit

**Pull request merged**